reminders/tests.py

```diff
@@ -72,13 +82,32 @@ def test_exist_reminder(self):
 
     def test_reminder_with_invalid_time(self, participant0, date_time=TIME_0):  # noqa: F811
         date_time = datetime(2022, 3, 22, 12, 12, 12, 0, tzinfo=timezone.utc)
-        expected = f'{time_format(date_time)} should be bigger than current.'
 
-        with pytest.raises(ValidationError, match=expected):
+        with pytest.raises(IntegrityError, match=PAST_DATE_TIME_ERROR):
             save_reminder(
                 create_reminder(participant0, METHOD_0, MESSAGES_0, date_time)
             )
 
-    def test_invalid_reminder_exist_twice(self, persist_reminder):
-        with pytest.raises(Exception, match=EXIST_REMINDER_ERROR):
-            persist_reminder.save()
+    def test_invalid_reminder_exist_twice(self):
+        event_participant = EventParticipant.objects.get(event_id__title="event1", user_id__username="testUser2")
+        event_date_time = datetime(2022, 8, 24, 13, 13, 13, 0, tzinfo=timezone.utc)
+        message = "Joined Meeting in 50 minutes"
```

Comment:
IMHO it was written too hard-coded,
I would prefer to use the same constants that you have you used in the persist method.
Another option, which I find better, is just to use the persist_reminder.save with pytest :)

Reply:
I have found that if there is an object that you jave already saved and try to save the same object twice it will not add any row to the db,
But if you create a new object with data that is the same as another row and try to save it, it will cause the error we are looking for.

You can try doing it in the shell and count the objects, the amount will be the same.

I will try to make it more readable but this is the way to check row duplication 😁

Reply:
done, used JOIN_MEETING instead of the text.
I have decided to leave the function as is because I want a specific variable to insert into the create_reminder method,
without the variables I have entered the exception will not be raised, and that will defeat the whole test